package.json

```diff
@@ -29,10 +29,6 @@
     "tiki"
   ],
   "scripts": {
-    "verify": "npm run verify:ios && npm run verify:android && npm run verify:web",
```

Comment:
while we don't move to TIKI License API, I think we should keep using the mobile version for manual testing, otherwise it will only work on automated tests with mocks